--- files/rkhunter~	2004-10-28 16:45:16.000000000 +0200
+++ files/rkhunter	2004-11-10 09:59:18.297957280 +0100
@@ -4267,7 +4267,7 @@
         VERSION=""
         case ${APPLICATION} in
           clamd)
-                VERSION=`${I}/clamd --version | grep 'ClamAV version' | awk '{ print $5 }'`
+                VERSION=`${I}/clamd clamd --version 2>&1 | cut -f 2 -d " " | cut -f 1 -d "/"`
                 ;;
 
           exim)
@@ -4283,11 +4283,7 @@
 #                VERSION=`${I}/mc -V | head -n 1 2> /tmp/mc.txt && cat /tmp/mc.txt | grep 'Midnight Commander' | sed 's/GNU Midnight Commander//' | awk '{ print $4 }' && rm -f /tmp/mc.txt`
 #                ;;
           named)
-                VERSION=`${I}/named -v | grep 'named' | grep -v '/' | awk '{ print $2 }'`
-                if [ ! "`echo ${VERSION} | grep "-"`" = "" ]
-                  then
-                    VERSION=`echo ${VERSION} | cut -d '-' -f1`
-                fi
+                VERSION=`${I}/named -v | awk '{ print $2 }'`
                 ;;
 	  openssl)
 		VERSION=`${I}/openssl version | head -n 1 | cut -d' ' -f2`
