diff -burN rkhunter.orig/files/rkhunter rkhunter/files/rkhunter
--- rkhunter.orig/files/rkhunter	2005-03-28 10:49:54.155301952 +0200
+++ rkhunter/files/rkhunter	2005-03-28 10:51:25.913352608 +0200
@@ -4411,11 +4411,7 @@
 #                VERSION=`${I}/mc -V | head -n 1 2> /tmp/mc.txt && cat /tmp/mc.txt | grep 'Midnight Commander' | sed 's/GNU Midnight Commander//' | awk '{ print $4 }' && rm -f /tmp/mc.txt`
 #                ;;
           named)
-                VERSION=`${I}/named -v | grep 'named' | grep -v '/' | awk '{ print $2 }'`
-                if [ ! "`echo ${VERSION} | grep "-"`" = "" ]
-                  then
-                    VERSION=`echo ${VERSION} | cut -d '-' -f1`
-                fi
+		VERSION=`${I}/named -v | awk '{print $2}'`
 		TEST=`${I}/named -v | grep 'named'`
 		logtext "Debug: ${TEST}"
                 ;;
